public/app/features/data-connections/DataConnectionsPage.test.tsx

```diff
@@ -37,7 +47,8 @@ describe('Data Connections Page', () => {
   test('shows the "Data sources" tab by default', async () => {
     renderPage();
 
-    expect(await screen.findByText('The list of data sources is under development.')).toBeVisible();
+    expect(await screen.findByText('Add data source')).toBeVisible();
```

Comment:
I'm likely missing something here but shouldn't we assert this is a button? Maybe the following can work:

```suggestion
    expect(await screen.findByRole('button', { name: /add data source/i }).toBeVisible()
```

Reply:
Nah, you are correct mate, this should be asserted to be a button, indeed 👍